react-components/stories/index.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import CopyTextAreaTest from './apps/details/CopyTextArea.stories'
+import ToolDetailsTest from './apps/details/ToolDetails.stories'
+import CategoryTreeTest from './apps/details/CategoryTree.stories'
+
+storiesOf('CopyTextArea', module).add('with test text', () => <CopyTextAreaTest/>);
+storiesOf('CategoryTree', module).add('with test hierarchy', () => <CategoryTreeTest logger={action('hierarchy')} />);
+storiesOf('ToolDetails', module).add('with test app', () => <ToolDetailsTest/>);
```

Comment:
Yes 👍 
I was already planning on something similar. I was going push a change like the following, that nests the stories under similar package structures as the source:
```javascript
storiesOf('apps/details/CategoryTree', module).add('with test hierarchy', () => <CategoryTreeTest logger={action('hierarchy')} />);
storiesOf('apps/details/ToolDetails', module).add('with test app', () => <ToolDetailsTest/>);
storiesOf('util/CopyTextArea', module).add('with test text', () => <CopyTextAreaTest/>);
```
This still allows for the possibility of more than 1 story per widget too (with different test data/views).

Although these stories only have 1 view and set of test data, so for these we can go with your suggestion:
```javascript
storiesOf('apps/details', module).add('CategoryTree', () => <CategoryTreeTest logger={action('hierarchy')} />);
storiesOf('apps/details', module).add('ToolDetails', () => <ToolDetailsTest/>);
storiesOf('util', module).add('CopyTextArea', () => <CopyTextAreaTest/>);
```
This also reminds me I should move `CopyTextArea` out of `apps/details/` 😊

Reply:
Ah ok cool, I didn't realize you could nest folders like that.  I like that idea 👍  And I hadn't considered multiple stories per widget, so that makes sense as well.